Grbl_Esp32/src/Probe.h

```diff
@@ -27,13 +27,13 @@
 const int PROBE_OFF    = 0;  // Probing disabled or not in use. (Must be zero.)
 const int PROBE_ACTIVE = 1;  // Actively watching the input pin.
 
+extern bool is_probe_away;
```

Comment:
This can go away since the only external access is via the setter function set_probe_direction